css-counter-styles-3/Overview.bs

```diff
@@ -2361,6 +2371,383 @@ Chinese: ''simp-chinese-informal'', ''simp-chinese-formal'', ''trad-chinese-info
 		</pre>
 	</div>
 
+<h4 id='extended-range-optional'>Extended Implementation (optional)</h4>
+
+	Some counter styles described in earlier chapters have been limited to an artifically small (though still useful) range
+	to reduce the overall complexity of the spec and the task of implementing those styles.
+	However, some implementations might consider the extra complexity worthwhile
+	for the additional range it offers to authors.
+	To accomodate this,
+	this section describes how to extend the limited counter-styles to a larger range.
+
+	This entire section is normative, but <strong>optional</strong>.
+	User-agents may ignore it and still be conformant.
+	If a user-agent implements some of the extended forms described in this section,
+	they must be implemented as described here.
+
+	The Chinese longhand styles are defined out to 10k with a [[#limited-chinese|specialized algorithm]],
+	while the Japanese and Korean longhand styles are defined similarly as ''additive'' styles.
+	However, these styles are defined out to 10<sup>16</sup> in common usage.
+	The following section describes an alternative algorithm for these styles.
```

Comment:
I think we can drop this paragraph. It's already explained in the section above.